gensim/sklearn_api/ldamodel.py

```diff
@@ -57,7 +57,7 @@ def fit(self, X, y=None):
         Calls gensim.models.LdaModel
         """
         if sparse.issparse(X):
-            corpus = matutils.Sparse2Corpus(X)
+            corpus = matutils.Sparse2Corpus(X, False)
```

Comment:
Named parameter better, for clarity. (here and elsewhere)

Reply:
Thanks! Done now.